front/app/containers/LandingPage/SignedOutHeader/HeaderContent.tsx

```diff
@@ -0,0 +1,205 @@
+import React from 'react';
```

Comment:
This is the text part of our layouts with left/center alignment options (layout 2, two columns, has the text left-aligned).